plenum/test/audit_ledger/test_audit_ledger_view_change.py

```diff
@@ -77,8 +78,10 @@ def patch_force_process_ordered():
 
         assert ordereds
         monkeypatch.setattr(slow_node, 'force_process_ordered', patch_force_process_ordered)
+        for n in txnPoolNodeSet:
+            n.start_catchup()
+        looper.run(eventually(lambda nodes: assertExp(all([n.mode == Mode.participating for n in nodes])), txnPoolNodeSet))
 
-    looper.run(eventually(lambda: assertExp(all(n.viewNo == 1 for n in txnPoolNodeSet))))
     ensureElectionsDone(looper=looper, nodes=txnPoolNodeSet)
```

Comment:
Why did we remove check for viewNo here?

Reply:
The test is checking work of force_process_ordered method. Maybe it should be renamed to "change_audit_on_force_ordered"?